src/com/hms_networks/americas/sc/config/ConfigFile.java

```diff
@@ -87,4 +88,14 @@ public void save() throws ConfigFileWriteException {
           e);
     }
   }
+
+  /**
+   * Gets a boolean representing the presence of the configuration file on disk.
+   *
+   * @return true if configuration file exists on disk
+   */
+  public boolean existsOnDisk() {
```

Comment:
```suggestion
   * Gets a boolean representing the presence of the configuration file in the file system.
   *
   * @return true if configuration file exists in the file system
   */
  public boolean exists() {
```

Reply:
I don't think it matters either way, but I changed it anyways.